en/faq.md

```diff
@@ -6,110 +6,126 @@ sidebarDepth: 2
+1. You didn't set up and configure the skin mod correctly.
+2. You have set up some mods(such as NonUpdate) that are interfering with the connection between the skin mod and LittleSkin.
+3. Your skin is cached by your skin mod,
+4. You are using the skin of Alex type, and you are playing Minecraft 1.8 or below.
 
-如果你有在服务器联机中加载材质的需求，并且你是服主，请在服务端也加载并正确配置 authlib-injector；否则请使用皮肤 Mod 加载材质。
+The solution is:
 
-### 为什么我无法使用 Universal Skin Mod 加载材质了？
+1. Set up and configure the skin mod correctly. You can learn how to do it by reading [Getting Started - Mod Configurations](/en/newbee/mod.html),
+2. Delete these mods, or let them bypass the connection between the skin mod and LittleSkin,
+3. Try again later. If it still doesn't update, please try to clean the cache of the skin mod by deleting the cache folder:
+    - The cache CustomSkinLoader is at `.minecraft/CustomSkinLoader/caches`；
+    - The cache SkinPort is at `.minecraft/cachedImages`
+4. Change the skin mod, or use the skin of Steve type:
+    - For Minecraft 1.7.10，please use SkinPort to load your skin,
+    - For the version below of Minecraft 1.7.10, please use the skin of Steve type.
 
-由于一些技术上的原因，LittleSkin 自 2020 年 1 月起不再支持 UniSkinAPI。你仍然可以通过传统加载方式加载材质，但我们建议使用 Universal Skin Mod 加载材质的用户尽快更换到 CustomSkinLoader。
+If you skin your problem is not caused by the reasons below, or it still doesn't work after tring the solutions, please read [How to report a problem](/en/report.html). Join the [user group](/en/user-group.html) or send [email tickets](/en/email.html) with the logs of the skin mod, or buy the one-by-one support service.
 
-### 启动器下载 authlib-injector 失败
+### I can sees my skin in single player game but not the muilt players game?
 
-虽然可能有多种报错，但一般都是网络原因引起的，请在启动器中更换下载源后再试。
+It often happens when you are playing on a server which is using Yggdrasil to load the authentication server.
 
-如果你使用 HMCL 3，如果更换下载源后仍然报错，请手动下载 authlib-injector，将其放入 `%appdata%\.hmcl\`（Windows）或 `$HOME/.hmcl/`（Linux 和 macOS）中并重命名为 `authlib-injector.jar`。使用其他启动器的用户请咨询启动器作者。
+Because of the skin loading system in Minecraft, in muilt player games, the server which opens the online mode can load the players' skins successfully. If the server you are playing load `authlib-injector` and set up LittleSkin's Yggdrasil API correctly, you can load your skin.
 
-### 别人看不到我的材质？
+If you need to load the players' skin in server, and you are the owner of the server, please set up `authlib-injector`, or you have to load your skins by Skin Mod.
 
-只有 **正确安装并配置** 皮肤 Mod 后才能加载来自 LittleSkin 的材质。
+### Why I can't use Universal Skin Mod to load my skin?
 
-如果你想让别人也看见你的材质，请让他们也正确安装并配置皮肤 Mod。
+For some technical reasons, LittleSkin stop the service of UniSkinAPI from the January in 2020. You can still use the legacy way to load your skin, but we suggest to use CustomSkinLoader instead of Universal Skin Mod.
```

Comment:
```suggestion
For some technical reasons, LittleSkin stop the support of UniSkinAPI in January 2020. You can still use the legacy way to load your skin, but we suggest to use CustomSkinLoader instead of Universal Skin Mod.
```

Reply:
Why this problem has NOT been addressed? /cc @Nofated095